packages/babel-preset-typescript/test/fixtures/opts/rewriteImportExtensions-createImportExpressions/output.mjs

```diff
@@ -0,0 +1,2 @@
+import("./a.js");
+import(babelHelpers.replaceTsImportExt(a));
```

Comment:
`a` might not be a string -- in that case we need to do `String(a)` first.

Reply:
I think `a + ""` should align to the `ToString` AO better than `String()`, since `String(symbol)` will return its description while `ToString(symbol)` is a type error.